src/loaders/jobs/taskfarmer/taskfarmer_task_mgr.py

```diff
@@ -0,0 +1,231 @@
+import datetime
```

Comment:
A couple behavior points I'm not sure about:

* the task manager is only going to know about the most recent job that ran for the coll/load_ver/tool combo when it started, or the most recent job that was run via `submit_job`. 
* If submit_job is called again for the same coll/load_ver/tool combo while jobs are running, things could get messy

Are those concerns we need to be worried about?

Reply:
if previous job is still running/pending, it will be caught by `_check_preconditions`.

Reply:
Maybe the manager should do the precondition check in submit_job?

Reply:
👍

Reply:
:+1: